grow/commands/build.py

```diff
@@ -18,8 +18,11 @@
 @click.option('--clear_cache',
               default=False, is_flag=True,
               help='Clear the pod cache before building.')
-@click.option('--file', '--pod_path', 'pod_paths', help='Build only pages affected by content files.', multiple=True)
-def build(pod_path, out_dir, preprocess, clear_cache, pod_paths):
+@click.option('--file', '--pod-path', 'pod_paths', help='Build only pages affected by content files.', multiple=True)
+@click.option('--display-untranslated',
```

Comment:
This is a great change. Thank you for working on this. Just a small nit on naming (know we talked about this but now that we thought of collecting untranslated messages into a PO file)...

Wonder if this flag should be simplified to `--show-untranslated` or `--collect-untranslated` (assuming it will eventually write the `untranslated.po` file?)